instances/technique/randomDotMotionStimulation.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/randomDotMotionStimulation",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "In a 'random dot motion stimulation' random dot(s) are simulated in an image and re-positioned at a new random location with each video frame [[Newsome & Paré, 1988](https://doi.org/10.1523/jneurosci.08-06-02201.1988).",
```

Comment:
suggestion (to actually make it a stimulation description):
"In a 'random dot motion stimulation' a subject is visually stimulated with a video where (a) simulated randomly distributed dot(s) are re-positioned at a new random location with each video frame [[Newsome & Paré, 1988](https://doi.org/10.1523/jneurosci.08-06-02201.1988)."